src/game_modes.h

```diff
@@ -0,0 +1,68 @@
+#ifndef _GAME_MODES_H_
+#define _GAME_MODES_H_
+
+#include "common.h"
+
+// These flags are mostly unused. Not enough context to discern their original purpose.
+enum GameModeFlags {
+    MODE_FLAG_NONE           = 0,
+    MODE_FLAG_INITIALIZED    = (1 << 0), // Set when the mod is initialized
+    MODE_FLAG_STEP_NOT_DONE  = (1 << 1), // Turned off after the first step is done
+    MODE_FLAG_3              = (1 << 2),
```

Comment:
We typically express unk flags by their bit values in hex, so these 3/4/5 would be 04/08/10